### PR TITLE
chore: try variant

### PR DESCRIPTION
**Pull request by Jeffxz**
Try to add a new varian - PWA on Android to see how it would look like

### CODE REVIEW

**Comment by Jeffxz** [comment]
This has dependency on https://github.com/w3c/epub-tests/pull/104

**Comment by Jeffxz** [comment]
If this way could work I am going to add PWA (iOS) as well.

File: reports/wysebee-android-pwa.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Wysebee",
+    "ref": "https://wysebee.com/",
+    "variant" : "PWA (Android)",
```
Comment (Jeffxz) [inline reports/wysebee-android-pwa.json:4]: hmm, or maybe Android (PWA) ?

File: reports/wysebee-android-pwa.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Wysebee",
+    "ref": "https://wysebee.com/",
+    "variant" : "Android(PWA)",
```
Comment (dlazin) [inline reports/wysebee-android-pwa.json:4]:
```suggestion
    "variant" : "Android (PWA)",
```

File: reports/wysebee-ios-pwa.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Wysebee",
+    "ref": "https://wysebee.com/",
+    "variant" : "iOS(PWA)",
```
Comment (dlazin) [inline reports/wysebee-ios-pwa.json:4]:
```suggestion
    "variant" : "iOS (PWA)",
```

**Review by dlazin — APPROVED** [review]
Excellent. This is what I was going to suggest based on your comment yesterday, but Wendy approved before I got around to it; thanks for fixing again.

File: reports/wysebee-android-pwa.json (hunk 2 above)
Comment (Jeffxz) [reply reports/wysebee-android-pwa.json]: updated

File: reports/wysebee-ios-pwa.json (hunk 3 above)
Comment (Jeffxz) [reply reports/wysebee-ios-pwa.json]: updated

**Pull request merged**